lua/neogit/status.lua

```diff
@@ -222,7 +227,7 @@ local function draw_buffer()
       })
     end
 
-    if git.branch.pushRemote_ref() then
+    if git.branch.pushRemote_ref() and git.repo.pushRemote.abbrev then
```

Comment:
I have this (possibly silly) idea to not directly access `lib.git.repo` state from outside `lib.git.*` modules. The thinking being to limit leaking the repo state abstraction into the "application" layer.

Reply:
That would be cool as well, I just added this when I got an nil error on an orphaned commit during rebasing

Reply:
The repo is already accesed when formatting the string just below

Reply:
Yeah.. Something to work towards ;)